tests/test_define.h

```diff
@@ -39,6 +39,15 @@ extern "C" {
 #define CURRENT_INDEX 1
 #define LAST_INDEX 2
 #define NONCE "THISTANGLEACCELERATORNONCES"
+#define MWM 14
```

Comment:
I think we have already defined `MWM` and `Threshould` in `config.h`. Maybe we don't have to define another one here.

Reply:
Already removed extra define.